internal/pkg/describe/lb_web_service.go

```diff
@@ -31,163 +31,163 @@ const (
 	noAdditionalFormatting = 0
 )
 
-// WebAppURI represents the unique identifier to access a web application.
-type WebAppURI struct {
-	DNSName string // The environment's subdomain if the application is served on HTTPS. Otherwise, the public load balancer's DNS.
-	Path    string // Empty if the application is served on HTTPS. Otherwise, the pattern used to match the application.
+// WebServiceURI represents the unique identifier to access a web service.
+type WebServiceURI struct {
+	DNSName string // The environment's subdomain if the service is served on HTTPS. Otherwise, the public load balancer's DNS.
+	Path    string // Empty if the service is served on HTTPS. Otherwise, the pattern used to match the service.
 }
 
-func (uri *WebAppURI) String() string {
+func (uri *WebServiceURI) String() string {
 	switch uri.Path {
-	// When the app is using host based routing, the app
-	// is included in the DNS name (app.myenv.myproj.dns.com)
+	// When the service is using host based routing, the service
+	// is included in the DNS name (svc.myenv.myproj.dns.com)
 	case "":
 		return fmt.Sprintf("https://%s", uri.DNSName)
-	// When the app is using the root path, there is no "path"
+	// When the service is using the root path, there is no "path"
 	// (for example http://lb.us-west-2.amazon.com/)
 	case "/":
 		return fmt.Sprintf("http://%s", uri.DNSName)
-	// Otherwise, if there is a path for the app, link to the
+	// Otherwise, if there is a path for the service, link to the
 	// LoadBalancer DNS name and the path
-	// (for example http://lb.us-west-2.amazon.com/app)
+	// (for example http://lb.us-west-2.amazon.com/svc)
 	default:
 		return fmt.Sprintf("http://%s/%s", uri.DNSName, uri.Path)
 	}
 }
 
 type serviceDiscovery struct {
-	AppName     string
-	ProjectName string
-	Port        string
+	Service string
+	App     string
+	Port    string
 }
 
 func (s *serviceDiscovery) String() string {
-	return fmt.Sprintf("%s.%s.local:%s", s.AppName, s.ProjectName, s.Port)
+	return fmt.Sprintf("%s.%s.local:%s", s.Service, s.App, s.Port)
 }
 
 type storeSvc interface {
-	GetEnvironment(projectName string, environmentName string) (*config.Environment, error)
-	ListEnvironments(projectName string) ([]*config.Environment, error)
+	GetEnvironment(appName string, environmentName string) (*config.Environment, error)
+	ListEnvironments(appName string) ([]*config.Environment, error)
 }
 
-type appDescriber interface {
+type svcDescriber interface {
 	Params() (map[string]string, error)
 	EnvOutputs() (map[string]string, error)
 	EnvVars() (map[string]string, error)
 	GetServiceArn() (*ecs.ServiceArn, error)
-	AppStackResources() ([]*cloudformation.StackResource, error)
+	ServiceStackResources() ([]*cloudformation.StackResource, error)
```

Comment:
nit: maybe just `StackResources()` ?

Reply:
Problem is we already have `StackResources` in `stack.go`